index.html

```diff
@@ -2,8 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="style.css">
     <title>JS Documentation</title>
-    <link rel="stylesheet" href="style.css">
+    

```

Comment:
Great Job so far👍✔✔. You have done well to follow all the instructions in the page. However, I recommend you make the side links to be navigable for more user experience. 
![image](https://github.com/Afsheen96/Trial-project/assets/99537435/f02b4edd-7cbc-407e-89e7-6bb4db6b7f63)